client/api/InnerClient.cs

```diff
@@ -353,27 +348,24 @@ public double NumberVariation(string key, Target target, double defaultValue)
             {
                 if (SdkInitialized) return evaluator.NumberVariation(key, target, defaultValue);
```

Comment:
while I'm being pedantic/distracted, in general methods are easier to read if the happy path is the main thing, and the conditional bits / edge cases are wrapped in conditions, i.e
```
if (!SdkInitialized) {
   // special case code
   // maybe return a default
   }
// normal handling
}
```

(and it seems you could write a geneic defaultValue log/compite fn that each method could call)

Reply:
Excellent advice.  I agree.  Done 👍  Will need another round of testing tomorrow, as I've made some decent changes to the public API.  We have a good integration framework that tests evaluations/default values to give us confidence.